x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_modal/index.tsx

```diff
@@ -107,13 +109,14 @@ export const AddExceptionModal = memo(function AddExceptionModal({
 }: AddExceptionModalProps) {
   const { http } = useKibana().services;
   const [comment, setComment] = useState('');
+  const { rule: maybeRule } = useRuleAsync(ruleId);
```

Comment:
I'm using this here to pull out the exception list details of the issue exception list. The error itself gives an `id` but no other info. Thought of reworking `useFetchOrCreateRuleExceptionList` to pull out the call it makes to the rule, but wanted to minimize changes for backport to 7.9.1.